devel-common/src/docs/build_docs.py

```diff
@@ -132,12 +132,15 @@ def _promote_new_flags():
     console.print("You can also use other extra flags to iterate faster:")
     console.print("   [bright_blue]--docs-only       - Only build documentation[/]")
     console.print("   [bright_blue]--spellcheck-only - Only perform spellchecking[/]")
-    console.print("   [bright_blue]--clean-build     - Refresh inventories for inter-sphinx references[/]")
     console.print()
     console.print("You can list all packages you can build:")
     console.print()
     console.print("   [bright_blue]--list-packages   - Shows the list of packages you can build[/]")
     console.print()
+    console.print("You can run clean build - refreshing inter-sphinx inventories or refresh airflow ones.\n")
+    console.print("   [bright_blue]--clean-build     - Refresh inventories for inter-sphinx references[/]")
+    console.print("   [bright_blue]--refresh-airflow-inventories - Force refresh airflow inventories.[/]")
```

Comment:
What's the difference between these two options? Reading this help text I'm not clear.

Reply:
We have:

* airflow intersphinx inventories
* other intersphinx inventories

The second option only cleans up airflow ones - I explained it more in the description of the PR:


> This includes both - airflow package inventory and external intersphinx docs as well.

> We are more concerned about availability of third-party inventories than ours, so we should use the cache mostly for the external packages rather than for ours and we should refresh it every time any dependency changes.

> The caching strategy implemented originally is to keep inventories for 12 hours, but this might cause 12 hours of failures in CI since we are using latest cached inventories for stability.

Reply:
Updated @ashb -> see if that strikes the right balance.